code.js

```diff
@@ -1,3 +1,27 @@
 function quicksort(array) {
-    return array;
+    var arr = [array];
+    var finished = false;
+    while (!finished) {
+        let tempArr = []
+        finished = true;
+        for (const item of arr) {
```

Comment:
You're iterating over something that you're changing at the same time here. This isn't great style and might break in unexpected ways. I would recommend to check whether your array is empty as the termination condition, and while it's not empty, pop items off it.

Reply:
I talked to you about this in office hours, and I think it's good now.  Is there anything else I'm required to do?

Reply:
Looks good.